addon/components/feature-result-item.js

```diff
@@ -0,0 +1,136 @@
+/**
+  @module ember-flexberry-gis
+*/
+
+import Ember from 'ember';
+import layout from '../templates/components/feature-result-item';
+
+/**
+  Component for display GeoJSON feature object details
+
+  @class FeatureResultItemComponent
+  @extends <a href="http://emberjs.com/api/classes/Ember.Component.html">Ember.Component</a>
+ */
+export default Ember.Component.extend({
+  /**
+    Flag: indicates where display detailed feature info
```

Comment:
Maybe you mean "Flag: indicates whether to display detailed feature info or not".